### PR TITLE
update rollup etc. and install the latest (transpiled) @terraformer/arcgis

### PR DESCRIPTION
**Pull request by jgravois**
thank you @destus90 for pointing out the problem with #1194. the error they saw in IE11 would have been caught out at compilation time if we had been running a newer version of rollup to bundle the library.

this PR is an alternative to #1199 that bumps @terraformer/arcgis to a version that is already transpiled and also updates rollup and its plugins to more modern versions so we don't miss similar errors in the future.

```js
john@zuzu esri-leaflet (master) $ npm run build

> esri-leaflet@2.4.0 prebuild /Users/john/code/ezree/esri-leaflet
> mkdirp dist


> esri-leaflet@2.4.0 build /Users/john/code/ezree/esri-leaflet
> rollup -c profiles/debug.js & rollup -c profiles/production.js


src/EsriLeaflet.js → dist/esri-leaflet-debug.js...

src/EsriLeaflet.js → dist/esri-leaflet.js...
created dist/esri-leaflet-debug.js in 908ms
  262 |    * Apache-2.0 */
  263 |
> 264 |   const edgeIntersectsEdge = (a1, a2, b1, b2) => {
      |  ^ Unexpected token: keyword «const»
  265 |     var uaT = (b2[0] - b1[0]) * (a1[1] - b1[1]) - (b2[1] - b1[1]) * (a1[0] - b1[0]);
  266 |     var ubT = (a2[0] - a1[0]) * (a1[1] - b1[1]) - (a2[1] - a1[1]) * (a1[0] - b1[0]);
  267 |     var uB = (b2[1] - b1[1]) * (a2[0] - a1[0]) - (b2[0] - b1[0]) * (a2[1] - a1[1]);
[!] (plugin uglify) Error: Unexpected token: keyword «const»
```
as you can see from the error, its only `uglify()` that was complaining about the ES2015 syntax. that's why @destus90 targeted the production build specifically.

either way, i feel bad for introducing that bug 🐛. i forgot that i'd sprinkled a little extra ES sugar when i ported arcgis-to-geojson-utils. ~~the good thing about @destus90's fix (and this one) is that they set you up to start using some more modern JS in this library and have it transpiled down for older browsers too if you ever feel like it.~~

i opened up this PR to get a little bit closer to the root of the problem, but also because i thought it would uncover the error to described in #1200.

i was only successful on the first front.

### CODE REVIEW

File: profiles/base.js [hunk 1]
```diff
@@ -26,7 +29,8 @@ export default {
     name: 'L.esri',
     globals: {
       'leaflet': 'L',
-      'esri-leaflet': 'L.esri'
+      'esri-leaflet': 'L.esri',
+      '@terraformer/arcgis': 'Terraformer'
```
Comment (jgravois) [inline profiles/base.js:33]: this solves an unrelated rollup nag.

```bash
Use output.globals to specify browser global variable names corresponding to external modules
/Users/john/code/ezree/esri-leaflet/node_modules/rollup/dist/rollup.js (guessing 'arcgis')
```

**Comment by patrickarlt** [comment]
Personally my preference would be to not transpile Esri Leaflet at all and resolve this in `@terraformer/arcgis`. `@terraformer/arcgis` should only ship ready-to-use already compiled code anyway so my preference would be to fix the build process there first to ensure Esri Leaflet can get dependencies it can use.

**Comment by jgravois** [comment]
> @terraformer/arcgis should only ship ready-to-use already compiled code anyway

i've been considering it a feature that @terraformer/arcgis ships both a UMD and a flat ESM module that can be tree shaken by consumers like Esri Leaflet.

it sounds like you're saying that the ESM just shouldn't need to be transpiled. i will make it so.

**Comment by jgravois** [comment]
alright, i just pushed another commit to revert the babel step here and 

what's left
* bump @terraformer/arcgis to the latest (complete with internal transpilation)
* bump rollup and friends to latest
* fix rollup nag

sorry this turned into a dogfooding session, but i appreciate you helping me improve terraformer@latest!

i don't have access to a box to test IE11 anymore, so help on that front would be appreciated.

**Pull request merged**